app/serializers/rest/instance_serializer.rb

```diff
@@ -33,6 +35,18 @@ def thumbnail
     end
   end
 
+  def icon
+    SiteUpload::ANDROID_ICON_SIZES.map do |size|
+      src = app_icon_path(size.to_i)
+      src = URI.join(root_url, src).to_s if src.present?
+
+      {
+        src: src || frontend_asset_url("icons/android-chrome-#{size}x#{size}.png"),
```

Comment:
Would it make sense to make this output consistent with the `thumbnail` property?

Reply:
We can, but there would be no `blurhash`, and what should the keys be? The thumbnail uses `@1x`, `@2x` but I dont think it makes sense for the icons. Should we use `size`?